src/utils/algorithm_specifiers.rs

```diff
@@ -0,0 +1,699 @@
+// Copyright (c) 2019, Arm Limited, All Rights Reserved
```

Comment:
We finally decided on a Copyright structure - feel free to follow the format in #62 . Same for the other new files in this PR

You can also go to https://github.com/parallaxsecond/parsec and add yourself/your company in the Contributors file, if you'd like!

Reply:
Fixed